### ISSUE TITLE
DATASET

### ISSUE THREAD

**Issue by lzzlxxlsz** [JasonZHM/CAEP#3]
what should I do if I want to use my own dataset ?

**Comment by haimengzhao** [comment]
Hi, thanks for your interest in our work.
If you want to change the training set, simply change the --dataset_path argument when calling train.py.
If you want to change the test set, you should first modify the parameters representing the height and width in utils.py/Kodak and save_kodak_img to be in line with your own dataset since it involves the decomposing and composing of patches. Then you should change the --testset_path argument when calling train.py.
